vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -30,6 +33,12 @@ const (
 	metricsNamespace = metrics.TopMetricsNamespace + "quality"
 )
 
+var (
+	// Both buckets are of length 17, 35 with prepended negative buckets
+	cpuBuckets    = getExponentialBuckets(0.01, 1000., 2.)
+	memoryBuckets = getExponentialBuckets(1e4, 1e12, 3.)
```

Comment:
I think we can safely start from 1e7 (that's only 10 MB, after all) and keep 2.0 growth factor.

Reply:
Done.